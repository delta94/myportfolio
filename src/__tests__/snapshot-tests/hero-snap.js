import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Hero from '../../components/hero';
import PrabhuLogo from '../mocks/prabhu-logo.json';
import ProfileImage from '../mocks/profile-image.json';
import HeroImage from '../mocks/hero-image.json';

it('Renders Hero correctly', () => {
  const wrapper = shallow(<Hero
    prabhuLogo={
    { sizes: PrabhuLogo }
  }
    profileImage={
    { sizes: ProfileImage }
  }
    heroImage={
    { sizes: HeroImage }
  }
  />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
