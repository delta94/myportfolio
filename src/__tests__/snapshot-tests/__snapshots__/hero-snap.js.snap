// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Hero correctly 1`] = `
<styled.section
  className="hero is-medium"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <styled.div>
    <Image
      Tag="div"
      alt=""
      className="hero-image-wrapper"
      fadeIn={true}
      outerWrapperClassName="hero-image-outer-wrapper"
      sizes={
        Object {
          "aspectRatio": 1.48720999405116,
          "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAwACBf/EABUBAQEAAAAAAAAAAAAAAAAAAAEC/9oADAMBAAIQAxAAAAEtc9YtYYf/xAAbEAACAgMBAAAAAAAAAAAAAAABAgASESEiMf/aAAgBAQABBQKy4uY2yDC3B9//xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAwEBPwEn/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAgEBPwE//8QAGhAAAgIDAAAAAAAAAAAAAAAAAAEQMQIhYf/aAAgBAQAGPwK8myzbh8j/xAAaEAEBAQEAAwAAAAAAAAAAAAABACERMXGR/9oACAEBAAE/IeukvA4Fhse+k3yQT0j13//aAAwDAQACAAMAAAAQ1+//xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAwEBPxAAT//EABYRAQEBAAAAAAAAAAAAAAAAAAEQEf/aAAgBAgEBPxB1Z//EABwQAQACAwADAAAAAAAAAAAAAAERIQBBUTGBwf/aAAgBAQABPxCcTBAHKvmXdYRkIKLw1Jpm5ek5BMsQ7MvyPeEgobDmf//Z",
          "sizes": "(max-width: 2500px) 100vw, 2500px",
          "src": "/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-2b4d0.jpg",
          "srcSet": "/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-98cf1.jpg 700w,
/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-90fd0.jpg 1400w,
/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-2b4d0.jpg 2500w",
        }
      }
    />
  </styled.div>
  <div
    className="hero-head"
  >
    <navbar
      prabhuLogo={
        Object {
          "sizes": Object {
            "aspectRatio": 1,
            "base64": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAAsSAAALEgHS3X78AAACXUlEQVQ4y2NgoAcw9oxlMPGMA7NBtIlHLJOJJwjDxBDyBAFIoakXVKNHLKOpVzwTTM7CN5HJGCgGYgPFQfL4DUJ2lZF7LAtMTscx3F7dNsQWxjd0i2EBGYjuG1RvQm0zQrIVaIiUgWv0LPvg9P8uEdn/9VwiZ8qa+EjB5I3cYwiEnUcczGscQIOLrPyS3/kll/2Pyqn5V9I88V9SSdt/u6C0t0CDiq0DUjiQ9WD1MlCSGWpzg11ozv+wjCqgYbW/I7Nr/hU3TfhX1Tntd2FD/3/HiPz/Bm7RTZBwjmPG6mVozDFDw6jfLiTzf0RW9Q8g/g8zsLRl0n8g/ukUnvPfwDVqAtQhuA00RhjYYxeSBTLwV0h6xb/wzKp/RY39YEOBXv/lFJ4LNDC6B24gNm+DBJG83G0bnPk/OLX8V3JJy7+emUv+zV2x8V9976x/QC//cgS7MLqbgAtBBsZCDYydYh2Y8b+yY+r3hWu2/lu2Yef/zXsO/9+w88C/1snzv9sEZfwHWjoJZiAo6eBKh8zQJJRl4ZfyP7m09X/f7GW/F6ze8nf5pp1/J85d/juxuOW/OVAOqCYdkg5BLozFlWxiQWkPlsbstB3DjzmEZv3Pr+8DYxAbmMiPAX0ATuRaDmG4syDMluSSZgZthzBGeC5xikgEhtdjYDJ5CmQnw8SBljHOWb4B7jvsXobmEmDihXsfBIAJXNLKP1kcSS2zY2gGLL/jLyhgeRqmCBgM4kC2GRA7A9kiyJYb4yscMJNRLMwCViDmAmIhYLCwIgcPyQCbV4guB3FphBWm6MUb3QAA78UJHRu0QDgAAAAASUVORK5CYII=",
            "sizes": "(max-width: 600px) 100vw, 600px",
            "src": "/static/prabhu-965ac803796b53b5fb4f84668bef7a53-a778c.png",
            "srcSet": "/static/prabhu-965ac803796b53b5fb4f84668bef7a53-c5352.png 150w,
/static/prabhu-965ac803796b53b5fb4f84668bef7a53-dbfa3.png 300w,
/static/prabhu-965ac803796b53b5fb4f84668bef7a53-a778c.png 600w,
/static/prabhu-965ac803796b53b5fb4f84668bef7a53-d1320.png 900w,
/static/prabhu-965ac803796b53b5fb4f84668bef7a53-7ae2a.png 1200w",
          },
        }
      }
    />
  </div>
  <div
    className="hero-body"
  >
    <ShapesView
      heroImage={
        Object {
          "sizes": Object {
            "aspectRatio": 1.48720999405116,
            "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAwACBf/EABUBAQEAAAAAAAAAAAAAAAAAAAEC/9oADAMBAAIQAxAAAAEtc9YtYYf/xAAbEAACAgMBAAAAAAAAAAAAAAABAgASESEiMf/aAAgBAQABBQKy4uY2yDC3B9//xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAwEBPwEn/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAgEBPwE//8QAGhAAAgIDAAAAAAAAAAAAAAAAAAEQMQIhYf/aAAgBAQAGPwK8myzbh8j/xAAaEAEBAQEAAwAAAAAAAAAAAAABACERMXGR/9oACAEBAAE/IeukvA4Fhse+k3yQT0j13//aAAwDAQACAAMAAAAQ1+//xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAwEBPxAAT//EABYRAQEBAAAAAAAAAAAAAAAAAAEQEf/aAAgBAgEBPxB1Z//EABwQAQACAwADAAAAAAAAAAAAAAERIQBBUTGBwf/aAAgBAQABPxCcTBAHKvmXdYRkIKLw1Jpm5ek5BMsQ7MvyPeEgobDmf//Z",
            "sizes": "(max-width: 2500px) 100vw, 2500px",
            "src": "/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-2b4d0.jpg",
            "srcSet": "/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-98cf1.jpg 700w,
/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-90fd0.jpg 1400w,
/static/rawpixel-603014-unsplash-08951b1aed4dc684f2a9ab00b49666ff-2b4d0.jpg 2500w",
          },
        }
      }
    />
    <div
      className="container"
    >
      <div
        className="columns is-centered is-multiline"
      >
        <div
          className="column is-12-desktop is-9-fullhd is-9-tablet"
          style={
            Object {
              "background": "rgba(255,255,255, 0.97)",
              "borderRadius": "4px",
              "boxShadow": "0 0 10px 1px rgba(0,0,0,0.5)",
              "padding": "0rem 0rem 0rem",
            }
          }
        >
          <div
            className="columns is-centered is-multiline is-gapless"
          >
            <div
              className="column is-3-desktop is-12-tablet"
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "justifyContent": "center",
                }
              }
            >
              <ProfileImage
                profileImage={
                  Object {
                    "aspectRatio": 0.7488888888888889,
                    "base64": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAbCAYAAAB836/YAAAACXBIWXMAAAsSAAALEgHS3X78AAAGtElEQVRIx0WW+VPU9xnH929p0yGJNh2TxvEaTGvaxKgzTZM6sWNi0yaphmYmHklERmtaohVRFOQWDNey3Cy7gBy77MIesCe7yx7csBxyUxBEp0lefb5fmukPn/nsfo/35/087+d5P19Nc+4lOku/wVJ1E2ttBj31mTgac3DWZ+NouEfAWoW/qxK/RYdPdp+5Eq9Zi9dUgaejHLcsV7uyyuhrL0VjKr+GRXeT7roMbI33cOpz6TPm427Ox2XMJWSrIWSvYcBRq+5BWzWBbjnEooBr8Qiwu1NZCngZGosuje6a29gaMnEacnC1FOBtK6K/sxhf232iffUM+wzEXI3qivY1MOCsE+Aa+gXYZ1EYV2wDC1tNd/Ut7ApYU7aA5eNrLyJofkCku4zxviqmQy2M+4wsDJuZiXQw6jcy5NUTkYNCjv+Dev7HVmOTvDn1Wbiac4XZfQLmb4n1CJhLR13230lP/oyzp05QfPMyPgntUczERKBFZR0RtkFJg7/nR6ZaNA5h1yvs3K35BEwPCFtLmAs2kJaSRMJPEtj1/A5e3fkSLz33PO8fOcKox8D8UBcTwVaGvQaVacBejd+qwyuiaZyNWSJCDt72QoJdD5jx19JYfJ2EhBc5c/g1Oi4cx3H7c2ylaeR/+VfGBHBt2s5stFNlOujRq4IpoXuFpaZXL+xa8kSEIjXUaW8tJ985ys9+mkDeB8eYzfiE1aqrEGlgpS2bQEsRa7O9LI1YmI60M9pvVFkGe6rxCUuNEq73YQEhc7GIoGPYoeWt13/Fjhd2knRwP8YPj1H86Xvorn7GZHkq1sJUlidtrMftzA+a1dAHPY0M2IWlCiiC+NoK1NyNCeBSxMDfPniXhOdeoOzUMcpPvMm5Q4l8uG83Naf/wK1PT7Ex62Rj2iksrUyH2xjxNRGWUvJL2Jo+yaFPBIlYSoj7apnxVJNz/C2OvPwKOe/8hkcZH7Ncf41IWhKR1D8Rq7nF1qKbTQFdHreK6h2MSSmFJex+CVvTp7+HXwCjAjjhqmJ1uJXKCx+RtPsVLkjIPR+/yePKi0SyvuTW0deIPfiaH1Z8bMz0sjrZzdxgJ+OBZrUBAgpgb2MmPhEl2iWAvToWo0YeuavJfP93fJG4h+zDiej/+Abp777B/eNH6Un9nC0RZVPW6oQADm3XZaxXhFFCdtbfwStlEzYVM+HUMisqP5k04a/O4IsDu7ly6ABfHdzH3WOH0J44SndGCpuPBHDKzuqolflYJ5P9Uj4/Atpq03E3ZRJqz2e0p5RpdxXrww8JaG9y+eAe3tu7l8O/fJXTv04k7e3f4tVl8p9FL2tj3ayIKHPRDib8zcSc9appaLoqvsFecwOf4S7RzkLifVoJv1RC/j15RxI5s38vJwX02C92ceWjk2wsuNiac7E2YWN52MLsQJuELIBS4EEpcI294a5QLSfm0EnZ1AlgHSFjIcmHD5G852XO7ttD0oH9vL1jJ23ld4ApvtsM8WzJw+acFPiElVFvEwPSzx6TuE1IUEfEUaaHOlmQmwui2qL8b866ynkp8HOJezm36+dcku5ZHeuCp2F+2BpgS5iuSNk8XuxjPGjEor1Nd+VtND6LlphPz/hAC49GzazEu/m3qLc2ZmVFLCvcXERN8hmmXPXwZIDv1vv5fiPAk3kXMwOtLMgh83Erva1FfCKHavqF6kBvHcP9BuKRh8wL6JI8tDzSxRMp3qfzfTyV0L5f8/Ns2cPTRRfPVrysy7245C4ebmVC3hv0N5F945IYbK00vNj6UMCgtlBcTp2NthEXE12L9/B4yiZseqU7XGzKvibFvC5usyThjot9DbkaGJL3lLRF7KJyR0m6+FkNEW8jg+5GxoTplJw4Jg8uDJvEqsQIZh3qWpuxsyKAixLF3IhZFSOszBtnrZq2oDi4pqXgGkG5EHbXE3U3qPau5HNM9tlYO0uTVnEXZQnQuEUVbl5SMjPcKa7dJEOsWh0FYXGcqJDSGHP+gZLHsHf7gpKL0VCzCjo12K4KNS9Ac2MW9bdSDfFY23behFW/jFdlAoak3MKeBmGY+0/sBplytir1YlQeUvIxKmtEkj4kBRsVa4oIk5jsI36DmppBJVwxBLu+ELfM4/4enfq+5t7ZP2OqyMCsvSMDPlcGe4WcqJXRmqXWVld5OqZv/4W55Ab2uhz8JhnyMi6VQW9vKsQsreiW4RSQ1gvJIZqiK0m05H5N/Z0U8i6eJvP8Xyi/fh79vRTMFWk4DHnyhVAiXw/bQI6mIgEqorngOmWpFyi/mYw+8zKmghRM+Rf5L597jKqlM4TZAAAAAElFTkSuQmCC",
                    "sizes": "(max-width: 337px) 100vw, 337px",
                    "src": "/static/profile-pic-5f63cea8412d899c907ab76de9bdcb4f-8ac9e.png",
                    "srcSet": "/static/profile-pic-5f63cea8412d899c907ab76de9bdcb4f-3608f.png 84w,
/static/profile-pic-5f63cea8412d899c907ab76de9bdcb4f-0b746.png 169w,
/static/profile-pic-5f63cea8412d899c907ab76de9bdcb4f-8ac9e.png 337w",
                  }
                }
              />
            </div>
            <div
              className="column is-9-desktop is-12-tablet"
            >
              <AboutMe />
            </div>
            <div
              className="column is-12-desktop is-12-tablet"
            >
              <Profile />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hero-foot"
  >
    placeholder
  </div>
</styled.section>
`;
